.github/GitReleaseManager/GitReleaseManager.yaml

```
@@ -18,6 +19,9 @@ issue-labels-alias:
   - name:    Bug
     header:  Bug Fix
     plural:  Bug Fixes
+  - name:    Breaking Change

```

```suggestion
  - name:    BreakingChange
```

I am as certain as I can be that this is what is needed here, otherwise I don't see how it will be picked up.